src/main/java/bio/terra/service/search/SearchService.java

```diff
@@ -133,29 +129,29 @@ public SearchQueryResultModel querySnapshot(
             int offset, int limit
     ) {
         //todo: move to bean for configuration, make injectable
-        final RestClientBuilder builder = RestClient.builder(new HttpHost("localhost", 9200));
-        try (RestHighLevelClient client = new RestHighLevelClient(builder)) {
-            SearchSourceBuilder searchSourceBuilder = new SearchSourceBuilder();
-            searchSourceBuilder.from(offset);
-            searchSourceBuilder.size(limit);
-            // see https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-wrapper-query.html
-            WrapperQueryBuilder wrapperQuery = QueryBuilders.wrapperQuery(searchQueryRequest.getQuery());
-            searchSourceBuilder.query(wrapperQuery);
-
-            SearchRequest searchRequest = new SearchRequest(indicesToQuery.toArray(new String[0]), searchSourceBuilder);
-
-            SearchResponse elasticResponse = client.search(searchRequest, RequestOptions.DEFAULT);
-            SearchHits hits = elasticResponse.getHits();
-            List<Map<String, String>> response = hitsToMap(hits);
-
-            SearchQueryResultModel result = new SearchQueryResultModel();
-            //do we want to include info on the index/snapshot the response came from?
-            result.setResult(response);
-            return result;
+        SearchSourceBuilder searchSourceBuilder = new SearchSourceBuilder();
+        searchSourceBuilder.from(offset);
+        searchSourceBuilder.size(limit);
+        // see https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-wrapper-query.html
+        WrapperQueryBuilder wrapperQuery = QueryBuilders.wrapperQuery(searchQueryRequest.getQuery());
+        searchSourceBuilder.query(wrapperQuery);
 
+        SearchRequest searchRequest = new SearchRequest(indicesToQuery.toArray(new String[0]), searchSourceBuilder);
+
+        SearchResponse elasticResponse;
```

Comment:
Can be `final` because it must be assigned only once before the `try` ends.
```suggestion
        final SearchResponse elasticResponse;
```